addons/udes_stock/wizard/stock_immediate_transfer_views.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="view_immediate_transfer" model="ir.ui.view">
+        <field name="name">stock.immediate.transfer.view.form</field>
```

Comment:
Suggestion: add `.udes_stock` at the end of name, just incase we need to debug the views for whatever reason.